docs/tutorials/column_filtering.ipynb

```diff
@@ -0,0 +1,130 @@
+{
```

Comment:
<p>Use second-level headings for sections. Looking at the region and row filter notebooks, the outline looks like</p><p> </p><ul><li>Title</li><li class="ql-indent-1">Introduction</li><li class="ql-indent-1">Load a catalog</li><li class="ql-indent-1">1 - interesting thing 1 (Load the catalog with all columns)</li><li class="ql-indent-1">2 - Interesting thing 2 (Load the catalog with a subset of columns)</li><li class="ql-indent-1">Close the client</li><li class="ql-indent-1">About</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>